Cargo.toml

```diff
@@ -6,14 +6,23 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0"
 clap = { version = "3.1", features = [ "derive" ] }
-dropshot = "0.6.0"
+progenitor = { git = "https://github.com/oxidecomputer/progenitor" }
+dropshot = { git = "https://github.com/oxidecomputer/dropshot" }
 serde = { version = "1.0", features = [ "derive" ] }
 slog = { version = "2.5.0", features = [ "max_level_trace", "release_max_level_debug" ] }
+slog-term = "2.7"
+slog-async = "2.7"
+slog-envlogger = "2.2"
 tokio = { version = "1.17", features = [ "full" ] }
 toml = "0.5"
 trust-dns-server = "0.21"
 trust-dns-proto = "0.21"
+trust-dns-resolver = "0.21"
 pretty-hex = "0.2.1"
 schemars = "0.8"
 sled = "0.34"
-serde_json = "1"
+serde_json = "1.0"
+reqwest = { version = "0.11", features = ["json", "rustls-tls", "stream"] }
```

Comment:
Should any of these be dev-dependencies instead?

Reply:
Yup, totally. I found a few that are only used in tests.

Reply:
Updated in 3c4acfa